### PR TITLE
 PERF-2181 LLT remove workloads

### PR DESCRIPTION
**Pull request by jimoleary**
Continuing to add more workloads.

### CODE REVIEW

File: src/workloads/scale/LLTRemove.yml [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+  Threads: 1
+  Database: *dbname
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20]
+      NopInPhasesUpTo: 20
+      PhaseConfig:
+        Repeat: 1
+
+- Name: UpdateBaseline.Low.50
```
Comment (dalyd) [inline src/workloads/scale/LLTRemove.yml:138]: Actor name needs update

**Review by dalyd — CHANGES_REQUESTED** [review]
The delete option part LGTM. Philosophical question of whether we would like changes like that in separate commit from the workload or not. No strong opinion from me. 

Same comments as query workload here  plus one more:
Do we need to insert more documents during this test? Will be run out of documents to remove during the run of this test? The number of documents should be decreasing across this test. How fast do they decrease?

File: src/workloads/scale/LLTRemove.yml (hunk 1 above)
Comment (jimoleary) [reply src/workloads/scale/LLTRemove.yml:138]: :+1:  plus the query names etc...

**Review by jimoleary — COMMENTED** [review]
> The delete option part LGTM. Philosophical question of whether we would like changes like that in separate commit from the workload or not. No strong opinion from me.
> 
In this case, I think it is ok to have this change with the workload. In general, I would probably do it this way anyway (just to keep the number of reviews down). 

> Do we need to insert more documents during this test? Will be run out of documents to remove during the run of this test? The number of documents should be decreasing across this test. How fast do they decrease?

Good question. I think for this test, we will probably need to put another load phase somewhere or add more documents in the initial load (or both). I may leave that to the evaluate ticket. At this stage, I want to get the workloads in place,

**Comment by jimoleary** [comment]
Note: I had gotten ahead of myself on the PR title (PERF-2181 is correct).

**Review by dalyd — APPROVED** [review]

**Comment by jimoleary** [comment]
@dalyd FYI I have a workload run that is just about to finish and there are ~18,000,000 docs per collection.

**Pull request merged**